users/tests/test_signup.py

```diff
@@ -106,4 +106,4 @@ def test_unauthenticated_user_view_signup(self, client):
     def test_signup_form_and_template_displayed(self, client):
```

Comment:
Line 29 shouldn't be `client.post(self.signup_url, data=valid_signup_details)`?

Reply:
Thanks! didn't notice it